examples/analog-clock.rs

```diff
@@ -124,14 +124,14 @@ fn draw_minute_hand(minute: u32) -> Styled<Line, PrimitiveStyle<BinaryColor>> {
 /// function can't be returned.
 fn draw_digital_clock<'a>(time_str: &'a str) -> impl Iterator<Item = Pixel<BinaryColor>> + 'a {
     let text = Text::new(&time_str, CENTER - Size::new(48, 48))
-        .into_styled(TextStyle::new(Font12x16, BinaryColor::Off));
+        .into_styled(MonoTextStyle::new(Font12x16, BinaryColor::Off));
 
     // Add a background around the time digits. Note that there is no bottom-right padding as this
     // is added by the font renderer itself
     let text_dimensions = text.bounding_box();
     let background = Rectangle::new(
         text_dimensions.top_left - Point::new(3, 3),
-        text_dimensions.size + Size::new(3, 3),
+        text_dimensions.size + Size::new(4, 4),
```

Comment:
This isn't directly related to this change, but I just noticed that this wasn't adjusted correctly when we changed way `Rectangle` is implemented.